final/problem-def-ana/problem-def.tex

```diff
@@ -1,10 +1,8 @@
 \section{Problem Definition}
 \label{sec:problem-definition}
 
-The previous sections provided the necessary background knowledge of
-the problem domain. Now that the background knowledge has been
-explained, this section will discuss the problem definition of the
-client.
+Now that the background knowledge has been explained, this section discusses the
+problem definition of the client.
```

Comment:
This is not what he was referring to there. What he means is to first have a general question, then go in depth on that question. This is precisely what is done in the next paragraph: first we state our main problem definition. Then we make sub-problems